Updated README.md

juanmaguitar opened the pull request:
I have updated the README.md with more accurate (and more clear) information

minaminfo commented:
As I told you I was busy doing other things
I tested your commit it's showing this error.Did you tested it before doing it?
```
Error: Cannot find module '/home/inam/web/gaunt-frontity/packages/gaunt-theme/frontity.settings'
Require stack:
- /home/inam/web/gaunt-frontity/node_modules/@frontity/file-settings/dist/src/importSettings.js
- /home/inam/web/gaunt-frontity/node_modules/@frontity/file-settings/dist/src/getSettings.js
- /home/inam/web/gaunt-frontity/node_modules/@frontity/file-settings/dist/src/index.js
- /home/inam/web/gaunt-frontity/node_modules/@frontity/core/dist/src/scripts/dev.js
- /home/inam/web/gaunt-frontity/node_modules/@frontity/core/dist/src/scripts/index.js
- /home/inam/web/gaunt-frontity/packages/gaunt-theme/node_modules/frontity/dist/src/commands/dev.js
- /home/inam/web/gaunt-frontity/packages/gaunt-theme/node_modules/frontity/dist/src/commands/index.js
- /home/inam/web/gaunt-frontity/packages/gaunt-theme/node_modules/frontity/dist/src/cli/index.js
```

juanmaguitar commented:
Well, this specific Pull Request only contains changes to a README.md so it shouldn't affect the project (it's just documentation)

Maybe there were some conflicts when you merged my previous PR?

My fork is here (the source of my PR's) → https://github.com/frontity-juanmaguitar/gaunt-frontity

It is working fine on my machine. You can clone it and try to see if it works

juanmaguitar commented:
Hi @minaminfo,

 there was an error with the dependencies of the project caused because I did an `npm install` inside the `packages/gaunt-theme` (that is not necessary, I updated the README.md with this info) which caused the creation of a `package-lock.json` in that folder that shouldn't exist there.

So I've added to this PR the removal of this file and the update of the  `package-lock.json` in the root folder.

This solves this error I was getting after doing a clean clone of my fork...

```
 ERROR in ./packages/gaunt-theme/src/components/loader.js
    Module not found: Error: Can't resolve 'react-spinners/ScaleLoader' in '/Users/juanma/PROJECTS/2020/FRONTITY/DEMOS/gaunt-frontity-juanmaguitar/packages/gaunt-theme/src/components'
    resolve 'react-spinners/ScaleLoader' in '/Users/juanma/PROJECTS/2020/FRONTITY/DEMOS/gaunt-frontity-juanmaguitar/packages/gaunt-theme/src/components'
      Parsed request is a module
      using description file: /Users/juanma/PROJECTS/2020/FRONTITY/DEMOS/gaunt-frontity-juanmaguitar/packages/gaunt-theme/package.json (relative path: ./src/components)
        Field 'browser' doesn't contain a valid alias configuration
        resolve as module

```

So, if you do a clean clone of  https://github.com/frontity-demos/gaunt-frontity you should have your theme working properly under the suggested structure.

If you merge this PR you should have in https://github.com/minaminfo/gaunt-frontity the same content than in https://github.com/frontity-demos/gaunt-frontity

Any doubt, don't hesitate in asking
